Quests/QuestNPC.cs

```diff
@@ -14,18 +15,18 @@ public class QuestNPC : GlobalNPC
 
         private BaseQuest availableQuest;
 
-        private int questCheckTick = 0;
+        private int questCheckTick = 300;
 
         private BaseQuest GetCurrentAvailableQuest(NPC npc)
         {
             if (availableQuest != null) return availableQuest;
             if (questCheckTick > 0)
             {
-                questCheckTick++;
-                if (questCheckTick >= 600) questCheckTick = 0;
+                questCheckTick--;
+                if (questCheckTick <= 0) questCheckTick = 0;
```

Comment:
This line just doesn't really do anything, I think. There isn't really any reason to reset questCheckTick here.